ai-ml/model-train/manifests/03-notebook/notebook.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels: &labels
+    app: jupyter-notebook
+  name: notebook
+spec:
+  ports:
+  - port: 8888
+  selector: *labels
+  #type: LoadBalancer
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: notebook
+  labels: &labels
+    app: jupyter-notebook
+spec:
+  selector:
+    matchLabels: *labels
+  template:
+    metadata:
+      annotations:
+        gke-gcsfuse/volumes: "true"
+      labels: *labels
+    spec:
+      serviceAccountName: bucket-access
+      containers:
+      - name: jupyter
+        image: quay.io/jupyter/pytorch-notebook:cuda11-python-3.11
+        #image: tensorflow/tensorflow:2.17.0-gpu-jupyter
+        #command: [ "/bin/bash", "-c", "--" ]
+        #args: [ "while true; do sleep 30; done;" ]
```

Comment:
**Question:** Is this (chunk of commented-out code) intentional?

Reply:
fixed